e2e-tests/cypress/integration/eligibility-local-planning-department.feature

```diff
@@ -3,14 +3,23 @@ Feature: A prospective appellant supplies a Local Planning Department for the ca
     Not every LPA is taking part in this iteration of the service.
 
   Scenario: Local Planning Department is required
+    Given the list of Local Planning Department is presented
     When the user does not provide a Local Planning Department
-    Then the user is informed that a Local Planning Department is required
+    Then the user is informed that a Local Planning Department in the provided list is required
+
+  Scenario: Local Planning Department not in the list
+    Given the list of Local Planning Department is presented
+    When the user provides a Local Planning Department not in the provided list
+    Then the user is informed that a Local Planning Department in the provided list is required
+    And appeal is not updated with the unknown Local Planning Department
 
   Scenario: Some Local Planning Departments are not participating in this service
+    Given the list of Local Planning Department is presented
     When the user provides a Local Planning Department that is not participating in this service
     Then the user is informed that the selected Local Planning Department is not participating in the service
-    And the user is directed to the Appeal a Planning Decision service
```

Comment:
are we deliberately losing the confirmation that we're pointing the user to the other service?

we -could- just wrap that confirmation up  in 'the user is informed that the selected LPD is not participating' step maybe if we're specifically trying to reduce our 'and' count? but i don't see that check happening anywhere in the PR, so feels like we either keep the existing line or wire its check into an existing step..

Reply:
Discussed in pm

Reply:
cool, happy we're not losing any coverage 👍 

cool from a QA perspective, assume we still need a review of the code, so not specifically ticking this as 'approved' as i'm not reading anything other than the e2e-tests section..